pkg/windows/windows.go

```diff
@@ -592,10 +592,21 @@ func (vm *windows) createDirectories() error {
 func (vm *windows) removeDirectories() error {
 	vm.log.Info("removing directories")
 	for _, dir := range RequiredDirectories {
+		// exclude WICD binary and credential files removal used in the WICD cleanup cmd
```

Comment:
You can remove `removal` from the comment, however the comment is misplaced. It should come inside the if statement.